### ISSUE TITLE
`1tbs` allows consecutive `if` statements

### ISSUE THREAD

**Issue by imnotjames** [eslint/eslint#10517]
**Tell us about your environment**

* **ESLint Version:** 4.19.1
* **Node Version:** v10.5.0
* **npm Version:** 6.1.0

**What parser (default, Babel-ESLint, etc.) are you using?**

Default.

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
// Expected invalid for 1tbs
if (true) {
  // Operation 1
} if (true) {
  // Operation 2
}

// Expected Good for 1tbs
if (true) {
  // Operation 1
}
if (true) {
  // Operation 2
}
```

```bash
eslint ./test.js
```

**What did you expect to happen?**

I expected an error because the expected invalid code does not conform to `1tbs`.

**What actually happened? Please include the actual, raw output from ESLint.**

No error was emitted.

**Comment by imnotjames** [comment]
As a temporary workaround, I've enabled the following rules:

* `max-statements-per-line` with a max of `1` line.
* `padding-line-between-statements` to enforce padding between anything and `if` statements.

**Comment by platinumazure** [comment]
This feels like a bug to me: The intent of 1tbs is that only else if statements would be allowed on the same line as the closing brace of a previous block.

That said, we have to be careful about classifying this since it could be a breaking change to some setups.

**Comment by platinumazure** [comment]
**TSC Summary:** This issue was reported as a bug. The complaint is that it is possible to do consecutive if statements in 1tbs brace-style. Arguably the intent of 1tbs is to only allow consecutive else if's, not consecutive ifs. Given how long the current behavior has been out there, I wonder if this should be considered a breaking change.

**TSC Question:** Should this be accepted as a bug, and if so, as a breaking change or non-breaking?

**Comment by not-an-aardvark** [comment]
In [today's TSC meeting](https://github.com/eslint/tsc-meetings/blob/430455a12366c4614f322734b38521ef12870251/notes/2018/2018-08-02.md#1tbs-allows-consecutive-if-statements), the TSC consensus was that this behavior doesn't really fall under the `brace-style` rule since it involves two separate statements, so we decided not to make a change here. I would recommend using [`max-statements-per-line`](https://eslint.org/docs/rules/max-statements-per-line) or a custom rule if you'd like to disallow this pattern in your codebase.
